backend/src/models/v2/Model.ts

```diff
@@ -13,6 +13,14 @@ export interface CollaboratorEntry {
   roles: Array<'owner' | 'contributor' | 'consumer' | string>
 }
 
+export interface ModelCardInterface {
+  schemaId: string
+  version: number
+  createdBy: string
```

Comment:
Is this going to be just the username or is it worth adding in the full entity object?

Reply:
We should always use `users` whenever it's an action done by someone.  There's never a 'group' that creates a release, it's always a specific person.  (Also, entities are now strings of the form `${type}:${id}`).

Reply:
Yes sorry I forgot the change in entity format, yeah I agree with this in that case.